src/main/java/net/snowflake/ingest/streaming/internal/OpenChannelRequestInternal.java

```diff
@@ -83,6 +89,10 @@ String getWriteMode() {
     return writeMode;
   }
 
+  boolean getIsIceberg() {
+    return isIceberg;
+  }
```

Comment:
pl add isIceberg to the getStringForLogging method too.

Reply:
Added